### ISSUE TITLE
Graphic dependencies are quite heavy

### ISSUE THREAD

**Issue by BeastyBlacksmith** [marcom/ViennaRNA.jl#8]
I don't know how essential the visualization functions are for users of this package, but loading the whole Makie stack regardless if you need that or not is quite expensive. For comparison (timings in a temporary environment after precompilation):
ViennaRNA@0.4:
```julia
julia> @time using ViennaRNA
  0.185636 seconds (747.30 k allocations: 48.920 MiB, 5.63% gc time, 6.60% compilation time)
```
ViennaRNA@0.5 (includes Luxor):
```julia
julia> @time using ViennaRNA
  0.700532 seconds (2.55 M allocations: 171.304 MiB, 5.47% gc time, 6.65% compilation time)
```
ViennaRNA@0.7 (includes CairoMakie):
```julia
julia> @time using ViennaRNA
  4.242130 seconds (14.88 M allocations: 1010.827 MiB, 9.37% gc time, 21.10% compilation time)
```

I'm happy to spend an Plots.jl recipe, if thats of interest until Makie has lightweight recipes itself, otherwise I'd prefer the visualization stuff to live in a separate package, such that you can use the other functionality without pulling a large amount of dependencies in.

**Comment by marcom** [comment]
I agree 100% that the load time is annoying. I did try adding precompile statements but didn't really get any improvements.

I think moving the plotting to a separate package makes sense. Maybe `DrawRNA.jl`?

This future `DrawRNA` package would then probably again suffer from slow loading. Would using a Plots.jl recipe or Makie lightweight recipes be able to reduce the package load time?

Also the first plot with `plot_structure_makie` (after the package has been loaded) takes about 50 seconds, any ideas on how to speed that up?
